test-integration/user.test.ts

```diff
@@ -163,4 +163,14 @@ describe('User class', () => {
             expect(userInfo.account_name).toEqual(await user.storage.accountName);
         })
     );
+    test(
+        'login() fails with userName does not exists',
```

Comment:
you have just copied this. Definition of Done not met. Please check DOD
https://github.com/Tonomy-Foundation/Tonomy-ID-SDK/pull/141/files#diff-7421f6b492312299f10e3bc4cabd18a7a3f6db8395d830f15f3b6fe8159777beR120